sqlalchemy_kusto/dbapi.py

```diff
@@ -39,12 +39,12 @@ def connect(
     cluster: str,
     database: str,
     msi: bool = False,
-    workload_identity: bool = False,
     user_msi: str | None = None,
+    workload_identity: bool = False,
     azure_ad_client_id: str | None = None,
     azure_ad_client_secret: str | None = None,
     azure_ad_tenant_id: str | None = None,
-):
+):  # pylint: disable=too-many-positional-arguments
```

Comment:
remove all pylint comments please

Reply:
Hello @pa1ch , done. Removed the lint hints